de.blobbyvolley.BlobbyVolley2.yaml

```diff
@@ -22,45 +25,25 @@ modules:
       - ./b2 install variant=release link=shared runtime-link=shared cxxflags="$CXXFLAGS" linkflags="$LDFLAGS" -j $FLATPAK_BUILDER_N_JOBS
     sources:
       - type: archive
-        url: https://dl.bintray.com/boostorg/release/1.74.0/source/boost_1_74_0.tar.bz2
-        sha256: 83bfc1507731a0906e387fc28b7ef5417d591429e51e788417fe9ff025e116b1
+        url: https://github.com/boostorg/boost/releases/download/boost-1.82.0/boost-1.82.0.tar.xz
+        sha256: fd60da30be908eff945735ac7d4d9addc7f7725b1ff6fcdcaede5262d511d21e
 
   - name: physfs
     buildsystem: cmake-ninja
     config-opts:
       - -DCMAKE_BUILD_TYPE=RelWithDebInfo
     sources:
       - type: archive
-        url: https://www.icculus.org/physfs/downloads/physfs-3.0.2.tar.bz2
-        sha256: 304df76206d633df5360e738b138c94e82ccf086e50ba84f456d3f8432f9f863
+        url: https://github.com/icculus/physfs/archive/refs/tags/release-3.2.0.tar.gz
+        sha256: 1991500eaeb8d5325e3a8361847ff3bf8e03ec89252b7915e1f25b3f8ab5d560
     cleanup:
       - /bin
 
   - name: blobbyvolley2
     buildsystem: cmake
+    build-commands:
+      - sed -i 's#<id>blobby</id>#<id>de.blobbyvolley.BlobbyVolley2</id>#g' linux/blobby.appdata.xml
     sources:
       - type: git
         url: https://github.com/danielknobe/blobbyvolley2.git
-        tag: v1.0
-      - type: patch
-        options: 
-          - '--binary'
-        paths: 
-          - c5329cbc50.patch
-          - b069d193b3.patch
-          - data-dir.patch
-          - data-dir-server.patch
-
-  - name: extra-files
-    buildsystem: simple
-    build-commands:
-      - install -D de.blobbyvolley.BlobbyVolley2.png /app/share/icons/de.blobbyvolley.BlobbyVolley2.png
-      - install -D de.blobbyvolley.BlobbyVolley2.desktop /app/share/applications/de.blobbyvolley.BlobbyVolley2.desktop
-      - install -D de.blobbyvolley.BlobbyVolley2.metainfo.xml /app/share/metainfo/de.blobbyvolley.BlobbyVolley2.metainfo.xml
-    sources:
-      - type: file
-        path: de.blobbyvolley.BlobbyVolley2.png
-      - type: file
-        path: de.blobbyvolley.BlobbyVolley2.desktop
-      - type: file
-        path: de.blobbyvolley.BlobbyVolley2.metainfo.xml
+        tag: v1.1
```

Comment:
1.1.1 is out and looks like upstream won't support migrating https://github.com/danielknobe/blobbyvolley2/issues/15#issuecomment-1650486768